ui/app/AppLayouts/Profile/popups/RemoveKeypairPopup.qml

```diff
@@ -0,0 +1,105 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.13
+import QtQml.Models 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups.Dialog 0.1
+import StatusQ.Core.Utils 0.1 as StatusQUtils
+
+import AppLayouts.Profile.controls 1.0
+import AppLayouts.Wallet 1.0
+
+import shared.panels 1.0
+import utils 1.0
+
+StatusDialog {
+    id: root
+
+    property bool isOnKeycard
+    property var relatedAccounts
+    property var getNetworkShortNames: function(chainIds){}
+
+    signal confirmClicked()
+
+    title: qsTr("Edit group name and image")
+    width: 521
+
+    ColumnLayout {
+        anchors.fill: parent
+        spacing: Style.current.xlPadding
+
+        StatusBaseText {
+            Layout.fillWidth: true
+            text: qsTr("Are you sure you want to remove Metamask seed phrase keypair? The keypair will be removed from all of your synced devices. Make sure you have a backup of your keys or seed phrase before proceeding.")
+            wrapMode: Text.WordWrap
+            font.pixelSize: 15
+        }
+
+        ColumnLayout {
+            Layout.fillWidth: true
+            Layout.bottomMargin: Style.current.padding
+            spacing: 8
+
+            StatusBaseText {
+                text: qsTr("Accounts related to this keypair will also be removed:")
+                font.pixelSize: 15
+            }
+
+            Rectangle {
+                Layout.fillWidth: true
+                Layout.preferredHeight: listview.height
+                color: Theme.palette.transparent
+                border.color: Theme.palette.baseColor2
+                border.width: 1
+                radius: 8
+                Column {
```

Comment:
@Khushboo-dev-cpp if there are more accounts, they won't be visible since this area is not scrollable.